### PR TITLE
Feature/mc 10755 list namespaces

### PR DESCRIPTION
**Pull request by y0geshdev**
#### Changes made

- Added docs for listing all namespace and getting details of a specific k8s namespace.

#### Related PRs
- https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/52
- https://github.com/cloudops/cloudmc-kubernetes-plugin/pull/24

### CODE REVIEW

**Review by m-orfanos — APPROVED** [review]

**Review by garrettkinman — APPROVED** [review]

File: source/includes/kubernetes/_k8_namespaces.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+
+<!-------------------- LIST NAMESPACES -------------------->
+
+### List namespaces
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/namespaces"
```
Comment (Shabirmean) [inline source/includes/kubernetes/_k8_namespaces.md]: Maybe
- `anenvironment` ---> `an-environment`
- `anenvironment` ---> `an_environment`

File: source/includes/kubernetes/_k8_namespaces.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+Retrieve a list of all namespaces in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the namespace.  
+`metadata` <br/>*object* | The metadata of the the namespace
+`metadata.annotations` <br/>*map* | The annotations of the namespace
+`metadata.creationTimestamp` <br/>*string* | The date of creation of the namespace as a string
+`metadata.name` <br/>*string* | The name of the namespace
+`metadata.resourceVersion` <br/>*string* | An opaque value that represents the internal version of namespace object
```
Comment (Shabirmean) [inline source/includes/kubernetes/_k8_namespaces.md]: internal version of **_"the"_** namespace

File: source/includes/kubernetes/_k8_namespaces.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+`metadata.selfLink` <br/>*string* | A URL representing the namespace object
+`metadata.uid` <br/>*object* | The UUID of the namespace
+`spec`<br/>*object* | The specification describes the attributes on a namespace.
+`spec.finalizers`<br/>*string* | An opaque list of values that must be empty to permanently remove object from storage
```
Comment (Shabirmean) [inline source/includes/kubernetes/_k8_namespaces.md]: remove **_"the"_** object

File: source/includes/kubernetes/_k8_namespaces.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+`status`<br/>*object* | The status information of the namespace
+`status.phase`<br/>*string* | The status of the namespace. Possible statuses are Active, Terminating and Unknown
```
Comment (Shabirmean) [inline source/includes/kubernetes/_k8_namespaces.md]: Maybe highlight the statuses with backtick markdown like:
`Active`, `Terminating`, `Unknown`

(\`)Status(\`)

File: source/includes/kubernetes/_k8_namespaces.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/namespaces/:id</code>
+
+Retrieve a namespace and all its info in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the namespace.
```
Comment (Shabirmean) [inline source/includes/kubernetes/_k8_namespaces.md:152]: Same comments as above for the List Namespaces

**Review by Shabirmean — APPROVED** [review]
Some minor suggestions.
LGTM!

File: source/includes/kubernetes/_k8_namespaces.md (hunk 5 above)
Comment (y0geshdev) [reply source/includes/kubernetes/_k8_namespaces.md:152]: 6b76e7d

File: source/includes/kubernetes/_k8_namespaces.md (hunk 4 above)
Comment (y0geshdev) [reply source/includes/kubernetes/_k8_namespaces.md]: 6b76e7d

File: source/includes/kubernetes/_k8_namespaces.md (hunk 3 above)
Comment (y0geshdev) [reply source/includes/kubernetes/_k8_namespaces.md]: 6b76e7d

File: source/includes/kubernetes/_k8_namespaces.md (hunk 2 above)
Comment (y0geshdev) [reply source/includes/kubernetes/_k8_namespaces.md]: 6b76e7d

File: source/includes/kubernetes/_k8_namespaces.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/kubernetes/_k8_namespaces.md]: 6b76e7d

**Pull request merged**